ecl/hql/hqlexpr.cpp

```diff
@@ -6614,6 +6615,27 @@ IHqlExpression *CHqlRecord::lookupSymbol(IIdAtom * fieldName)
     return ret;
 }
 
+IHqlExpression * CHqlRecord::lookupNearestSymbol(IIdAtom * fieldName)
```

Comment:
lookupNearestSymbol(IIdAtom * fieldName) is part of IHqlSimpleScope which CHqlRecord implements.